src/server/schema/resolvers/mutations/createLife.ts

```diff
@@ -0,0 +1,31 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext, Life } from '../../../database';
+import { GraphQLMutationResolvers } from '../definitions';
+
+const mutation: GraphQLMutationResolvers['createLife'] = async (
+    root,
+    { title, firstname, lastname, description, birthday, hobbies },
+    { getTranslations }
+) => {
+    const { collections } = await getDatabaseContext();
+
+
+    const newID = new ObjectId();
+
+    const document: Life = {
+        _id: newID,
+        title,
+        firstName: firstname,
+        lastName: lastname,
+        birthday: new Date(birthday * 1),
```

Comment:
if you use `DateTime` on GraphQL, then `birthday` here should be a native date already